### PR TITLE
KML fixes

### LINKED ISSUE #3544: Fix tests
The test https://github.com/geoadmin/mf-geoadmin3/blob/master/test/specs/map/KmlService.spec.js#L694 should have failed with the update of OpenLayers. 

If you execute this test alone (using `it.only(...)`) it fails as expected, but executing all the tests it succeeds.

### PR DESCRIPTION
**Pull request by oterral**
Improve the KML hacks using the old behavior (using default style values).

Fix #3544 .

[Test](https://mf-geoadmin3.int.bgdi.ch/teo_fixes/index.html?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&X=190370.14&Y=604667.85&zoom=7)

### CODE REVIEW

**Comment by oterral** [comment]
jenkins test

**Comment by oterral** [comment]
Please a review I need for import tool. @loicgasser @gjn

**Comment by gjn** [comment]
On it.

File: src/components/map/KmlService.js [hunk 1]
```diff
@@ -33,6 +33,23 @@ goog.require('ga_urlutils_service');
...
 
+      // Create the parser/writer KML
+      var setKmlFormat = function() {
+        if (!kmlFormat) {
+          // TO FIX
+          // Hack for #3531: Should be fix with next version of ol >3.18.2
+          // We create an empty format first to create the default style
+          // variables.
+          // https://github.com/openlayers/ol3/blob/master/src/ol/format/kml.js#L143
+          ol.format.KML();
```
Comment (gjn) [inline src/components/map/KmlService.js]: Wow. That's really ugly. Mental hint to remove it on next ol3 update.

**Pull request merged**